ops/ccs-ops-misc/synthetic-data/scripts/synthea-manual/pre-validate-synthea.py

```diff
@@ -0,0 +1,258 @@
+#
```

Comment:
Is this even used??? Isn't the validation in the Pipeline now?

Reply:
It is, but if someone for any reason wanted to manually use this script, it'll be there (for instance if they manually generated synthea data, they wouldnt have added the pre-validation data; alternatively, if there was a DB issue and they want to ensure they properly cleaned up so that the end state is valid again)